db/migrate/20220425230907_add_disabled_at_to_spree_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisabledAtToSpreeUsers < ActiveRecord::Migration[6.1]
+  def up
```

Comment:
```suggestion
  def change
```

And then Rails knows how to revert the migration as well.